packages/picasso.js/src/core/chart-components/axis/axis-arc-node.js

```diff
@@ -0,0 +1,34 @@
+import extend from 'extend';
+
+function appendStyle(struct, buildOpts) {
+  extend(struct, buildOpts.style);
+}
+
+export default function buildArcLine(buildOpts) {
+  const rect = buildOpts.innerRect;
+  const centerPoint = { cx: rect.width / 2, cy: rect.height / 2 };
+  const plotSize = Math.min(rect.height, rect.width) / 2;
+
+  const innerRadius = plotSize * buildOpts.radius;
```

Comment:
Should have the same formula as above:
const innerRadius = plotSize * (buildOpts.radius ?? 0.5);